spec/requests/api/v1/articles_spec.rb

```diff
@@ -13,8 +14,98 @@
+will be ordered by descending popularity.This parameter can be used in conjuction with `top`.",
+                  schema: { type: :string },
+                  example: "discuss"
+        parameter name: :tags, in: :query, required: false,
+                  description: "Using this parameter will retrieve articles with any of the comma-separated tags.
+Articles will be ordered by descending popularity.",
+                  schema: { type: :string },
+                  example: "javascript, css"
+        parameter name: :tags_exclude, in: :query, required: false,
+                  description: "Using this parameter will retrieve articles that do _not_ contain _any_
+of comma-separated tags. Articles will be ordered by descending popularity.",
+                  schema: { type: :string },
+                  example: "node, java"
+        parameter name: :username, in: :query, required: false,
+                  description: "Using this parameter will retrieve articles belonging
+            to a User or Organization ordered by descending publication date.
+            If `state=all` the number of items returned will be `1000` instead of the default `30`.
+            This parameter can be used in conjuction with `state`.",
+                  schema: { type: :string },
+                  example: "ben"
+        parameter name: :state, in: :query, required: false,
+                  description: "Using this parameter will allow the client to check which articles are fresh or rising.
+            If `state=fresh` the server will return fresh articles.
+            If `state=rising` the server will return rising articles.
+            This param can be used in conjuction with `username`, only if set to `all`.",
+                  schema: {
+                    type: :string,
+                    enum: %i[fresh rising all]
+                  },
+                  example: "fresh"
+        parameter name: :top, in: :query, required: false,
+                  description: "Using this parameter will allow the client to return the most popular articles
+in the last `N` days.
+`top` indicates the number of days since publication of the articles returned.
+This param can be used in conjuction with `tag`.",
+                  schema: {
+                    type: :integer,
+                    format: :int32,
+                    minimum: 1
+                  },
+                  example: 2
+        parameter name: :collection_id, in: :query, required: false,
+                  description: "Adding this will allow the client to return the list of articles
+belonging to the requested collection, ordered by ascending publication date.",
+                  schema: {
+                    type: :integer,
+                    format: :int32
+                  },
+                  example: 99
+
+        response "200", "A List of Articles" do
+          let(:"api-key") { "valid" }
```

Comment:
This sets the `api-key` header that the V1 API requires. It doesn't actually matter what string we pass here: rswag will be setting the response code, not our actual code.